_ruby_libs/pages.rb

```diff
@@ -361,6 +361,22 @@ def write(dest)
   end
 end
 
+class RelocatableStaticFile < Jekyll::StaticFile
+  def initialize(site, base, dir, name, dest = nil)
+    super(site, base, dir, name)
+    @dest = dest
+  end
+
+  def destination(dest)
```

Comment:
What's the difference between the instance variable `@dest` and the provided argument? Using the same name for both make it hard to see what's happening here.

Reply:
So, `dest` is the whole site destination directory while `@dest` is the static file location relative to the former. I've rewritten the code for clarity. PTAL.